fabfile.py

```diff
@@ -142,6 +151,9 @@ def install():
         run('mkdir media')
         run('chown www-data:www-data media')
 
+    # Export Sentry DSN key
+    run('echo "export VOLONTULO_SENTRY_DSN='VOLONTULO_SENTRY_DSN' >> ~/.bash_profile')
```

Comment:
Does it put the value of `VOLONTULO_SENTRY_DSN` variable or a string "VOLONTULO_SENTRY_DSN"? Looks like `f'` is missing in front of it.

Reply:
Not able to test it, so I will put `f'` before